### PR TITLE
Increase epsilon in MultiLinearSpanInterpolation test

### PR DESCRIPTION
**Pull request by knelli2**
## Proposed changes

Found from [this](https://github.com/sxs-collaboration/spectre/actions/runs/9899423044/job/27348289155?pr=6096) CI, increase the epsilon. First commit doesn't change the test at all, just makes formatting conform to our standards.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: tests/Unit/NumericalAlgorithms/Interpolation/Test_MultiLinearSpanInterpolation.cpp [hunk 1]
```diff
@@ -125,24 +127,28 @@ void test() {
...
 
-  auto y_interpolated = uniform_intp.interpolate(which_var, x);
-  auto y_interpolated_gen = general_intp.interpolate(which_var, x);
+  auto y_interpolated = uniform_intp.interpolate(which_var, point);
+  auto y_interpolated_gen = general_intp.interpolate(which_var, point);
 
+  const double epsilon = 1.0e-12;
+  CAPTURE(epsilon);
   for (size_t nv = 0; nv < NumVar; ++nv) {
+    CAPTURE(y_expected[nv]);
```
Comment (nilsdeppe) [inline tests/Unit/NumericalAlgorithms/Interpolation/Test_MultiLinearSpanInterpolation.cpp]: looks like clang-tidy wants these using gsl::at too (or some subset of them at least)

**Review by nilsdeppe — DISMISSED** [review]
LGTM, did you want to do the last few clang-tidy things? Up to you :)

File: tests/Unit/NumericalAlgorithms/Interpolation/Test_MultiLinearSpanInterpolation.cpp (hunk 1 above)
Comment (knelli2) [reply tests/Unit/NumericalAlgorithms/Interpolation/Test_MultiLinearSpanInterpolation.cpp]: Oops, my bad. Squashed those in as well.

**Review by nilsdeppe — APPROVED** [review]
Thank you!!! :D

**Pull request merged**